tests/OmniSharp.MSBuild.Tests/WorkspaceInformationTests.cs

```diff
@@ -122,6 +122,25 @@ public WorkspaceInformationTests(ITestOutputHelper output)
             }
         }
 
+        [Fact]
+        public async Task TestProjectWithReferencedProjectOutsideOfOmniSharp()
+        {
+            using (var testProject = await TestAssets.Instance.GetTestProjectAsync("TwoProjectsWithSolution"))
```

Comment:
What log level are we running OmniSharp at in these tests? If it's not "Debug", the logging isn't getting tested.

Reply:
When testing, we use a logger that captures *all* logging, regardless of level. When there's a failure, you get a dump of all logged messages.

Reply:
Also, FWIW, the logging *does* get tested in the sense that the call to the logger certainly happens. Whether or not we output it or not is a different concern.

Reply:
Ah, ok. Cool.